stream_srtp.go

```diff
@@ -113,17 +113,32 @@ func (r *ReadStreamSRTP) GetSSRC() uint32 {
 // WriteStreamSRTP is stream for a single Session that is used to encrypt RTP
 type WriteStreamSRTP struct {
 	session *SessionSRTP
+	buffer  []byte
```

Comment:
Is this thread safe? `WriteStreamSRTP` all of the `Stream` stuff should be.

Reply:
It's not thread-safe. I wasn't sure if it was a requirement, but sure I can change it.

Reply:
yea if you don't mind!

I think we need to make everything thread safe by default, just to avoid the sharp edges. I made that mistake at first and it bit a lot of people :(